app/controllers/partner_users_controller.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_String_literal: true
+
+class PartnerUsersController < ApplicationController
+  before_action :set_partner, only: %i[index create destroy resend_invitation]
```

Comment:
We only link to this controller from bank admins, but we don't here restrict access to only admins. I'll open a separate ticket to address this minor security flaw.

Reply:
Created #4458 to address